modules/repository-s3/src/yamlRestTest/java/org/elasticsearch/repositories/s3/RepositoryS3RegionalStsClientYamlTestSuiteIT.java

```diff
@@ -21,10 +21,11 @@ public class RepositoryS3RegionalStsClientYamlTestSuiteIT extends AbstractReposi
     @ClassRule
     public static ElasticsearchCluster cluster = ElasticsearchCluster.local()
         .module("repository-s3")
-        .configFile("repository-s3/aws-web-identity-token-file", Resource.fromClasspath("aws-web-identity-token-file"))
-        .environment("AWS_WEB_IDENTITY_TOKEN_FILE", System.getProperty("awsWebIdentityTokenExternalLocation"))
-        // The AWS STS SDK requires the role and session names to be set. We can verify that they are sent to S3S in the
-        // S3HttpFixtureWithSTS fixture
```

Comment:
Is this comment no longer relevant?

Reply:
It was never correct - we don't ever actually send any requests to S3 in this test suite, we're just doing some basic initialization to make sure that ES doesn't encounter any problems with e.g. the security manager.